reviews/utils.py

```diff
@@ -253,6 +253,27 @@ def notify_secretary(decision):
     send_mail(subject, msg_plain, settings.EMAIL_FROM, [secretary.email])
 
 
+def notify_supervisor_nogo(proposal, decision):
```

Comment:
The proposal parameter is a bit redundant. You can access that through ``decision.review.proposal``

It's not bad or anything, but it's more consistant to use that method, as it's also used for the ``notify_secretary`` function (in the template)